crates/ra_parser/src/syntax_kind/generated.rs

```diff
@@ -241,6 +241,102 @@ pub enum SyntaxKind {
 }
 use self::SyntaxKind::*;
 
+#[macro_export]
+macro_rules! tkn {
```

Comment:
Let's name this `TOKEN`: this generates a const, so it makes sense for this to be capitalized

So that one would use `TOKEN![,]`

Reply:
Hm, it'll be to screamy for short tokens though... Let's make it just

```
T![,]
```

?

Reply:
Maybe T! is too short? TKN![,] ?

Reply:
I  like `TKN` less than either of `T` and `TOKEN`. Dropping vowels is not very readable to me.

My personal preference would be

```
T, TOKEN # somewhat equivalent
TOK  
TKN
```

though, I don't have a strong opinion here and changing later would be easy

Reply:
TOKEN![,] is good for me, as IDE should be able to auto-complete the remain “KEN” after i type in “TO”

Reply:
I like T! most of all too.  I've made so.